SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1842,12 +1855,21 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			RPCService->AdvanceView();
 		}
 
+		if (DebugCtx != nullptr)
+		{
+			DebugCtx->AdvanceView();
+		}
+
+		if (ActorSystem.IsValid())
+		{
+			ActorSystem->Advance();
+		}
+
 		{
 			SCOPE_CYCLE_COUNTER(STAT_SpatialProcessOps);
 			Dispatcher->ProcessOps(GetOpsFromEntityDeltas(Connection->GetEntityDeltas()));
 			Dispatcher->ProcessOps(Connection->GetWorkerMessages());
 			CrossServerRPCHandler->ProcessMessages(Connection->GetWorkerMessages(), DeltaTime);
-			Receiver->ProcessActorsFromAsyncLoading();
```

Comment:
What happens to async loading? Disabled for now?
Or is it a regression from a fix that was done to separate async loading to collect loaded packages in an intermediate array?

Reply:
yep, disabled. async loading won't work until https://improbableio.atlassian.net/browse/UNR-4198 is done

Reply:
Was there an async loading gym/test @ImprobableNic?